src/autofill-utils.js

```diff
@@ -392,6 +392,14 @@ const wasAutofilledByChrome = (input) => {
     }
 }
 
+/**
+ * Checks if we should log debug info to the console
+ * @returns {boolean}
+ */
+function shouldLog () {
+    return window.sessionStorage?.getItem('ddg-autofill-debug') === 'true'
```

Comment:
This is a simple way to set the flag on all platforms. In the future we may make this more advanced with platform-specific strategies, but this works fine for now. If you have better ideas let me know.